ironfish/src/network/peers/peerManager.ts

```diff
@@ -535,10 +535,13 @@ export class PeerManager {
 
   /**
    * Returns true if the total number of connected peers is less
-   * than the target amount of peers
+   * than the target and max amount of peers
    */
   canCreateNewConnections(): boolean {
-    return this.getPeersWithConnection().length < this.targetPeers
+    return (
+      this.getPeersWithConnection().length < this.targetPeers &&
```

Comment:
Looks good! Can we just reuse this value so the function is not called twice? 
```typescript
const currentPeers = this.getPeersWithConnection().length
```

Reply:
@danield9tqh yup, updated.